cronjob.py

```diff
@@ -0,0 +1,81 @@
+
+from lib.modules.networkingmodule.networkingresponse import *
+import requests
+import json
+import numpy as np
+import os
+from dotenv import load_dotenv
+from pathlib import Path
+from aiohttp import ClientError
+from lib.modules.databasemodule.database import database
+db = database.instance()
+
+env_path = Path('.') / '.env'
+load_dotenv(dotenv_path=env_path)
+
+if __name__ == '__main__':
+    try:
+        header = {'Authorization': 'Bearer ' + os.getenv("SLACK_BOT_TOKEN")}
+        # Gets list of users in Networking channel
+        data = {'channel': 'C03QSC88Y4E'}
+        rawusers = requests.get('https://slack.com/api/conversations.members', headers=header, params=data).content
+        if not json.loads(rawusers.decode('utf-8'))['ok']:
+            print("Error: " + json.loads(rawusers.decode('utf-8'))['error'])
+        users = json.loads(rawusers.decode('utf-8'))['members']
+        users.remove('U03QC4N6TDF')
+        # Determines if users are alum
+        alum = []
+        student = []
+        for user in filter(lambda x: x != 'U03THUJ5Y2X', users):
```

Comment:
Once again bc this value is hardcoded and used twice, put it in some constant like BOT_ID bc then reading the code is easier